build/webpack.docs.js

```diff
@@ -29,21 +44,16 @@ const config = merge(baseConfig, {
     new webpack.DefinePlugin({
       'process.env.NODE_ENV': JSON.stringify('production'),
       VERSION: JSON.stringify(options.version)
-    }),
-
-    // Minify with dead-code elimination
-    new webpack.optimize.UglifyJsPlugin({
-      compress: {
-        warnings: false
-      }
     })
   ]
 });
 
-
-// First item in module.rules array is Vue
-config.module.rules[0].options.loaders = {
```

Comment:
I think this is causing the Less in the docs Vue files not to be loaded. I tried running `npm run dev` and the docs pages are wonky looking.

Reply:
No, this line is a right change. By the way, I fixed, now it should work!